src/style.css

```diff
@@ -1 +1,81 @@
-/* Styles go here */
+html {
+  --link-hover-active: #00acdc;
+}
+
+body,
+div,
+ul,
+li,
+a {
+  padding: 0;
+  margin: 0;
+}
+
+.header {
+  max-width: 1200px;
+  display: flex;
+  align-items: center;
+}
+
+.logo-link {
+  display: block;
+  width: 40px;
+  height: 40px;
+  margin-left: 50px;
```

Comment:
Please add padding for header Istead of using left/right margin for logo and nav. Should be smth like this:
![image](https://github.com/mate-academy/layout_moyo-header/assets/54936044/92483177-a03c-4695-83c6-cf5b4f77ad47)